cmd/clusterAddExternalWorker.go

```diff
@@ -142,6 +143,8 @@ An external server must meet the following requirements:
 
 		FatalOnError(err)
 
+		cluster.Nodes = append(cluster.Nodes, externalNode)
```

Comment:
this statement already comes 4 lines later. I save the new state as soon as the first changing operation succeeded, as ProvisionNodes()

Reply:
Yep just realised how the wait mechanism works, it was not working :) was refactoring to add a proper function and wait for the installation to actually happen, maybe I dont need to anymore 👍

Reply:
Yeah you just can remove this one